.gitignore

```diff
@@ -2,6 +2,7 @@
 .tags*
 /.idea/
 /browser/resources/brave_extension/
+/browser/resources/brave_webtorrent/*.bundle.js
```

Comment:
we shouldn't be writing back into the source directory. Please see the wip sync branch

Reply:
It seems we couldn't access these *.bundle.js files from the extension if it's in gen, so I'm still generating webpack output files into src, they're now under `components/brave_webtorrent/entension/out`.